Directory.Build.props

```diff
@@ -3,8 +3,7 @@
   <PropertyGroup>
     <!-- Fixes https://github.com/dotnet/maui/pull/12114 -->
     <PublishReadyToRun>false</PublishReadyToRun>
-
-    <LangVersion>latest</LangVersion>
+    <LangVersion>preview</LangVersion>
```

Comment:
can this break our users somehow? I would suggest to keep it at latest, and open an proposal for this change

Reply:
LangVersion latest uses c# 10, but we have features from c# 11

Reply:
C#11 is already stable